plugin/tests/mutagen/update-all-references.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
```

Comment:
I've borrowed this from clippy. I'm not sure if I can borrow it as-is.

Reply:
I think I can get a specific license for use in mutagen.